changelog/58708.md

```diff
@@ -0,0 +1,3 @@
+**fixed:**
+
+Code maintenance replacing usage of `botomod._exactly_one` with `salt.utils.data.exactly_one`.
```

Comment:
```suggestion
Replaced usage of `botomod._exactly_one` with `salt.utils.data.exactly_one`.
```

Reply:
Submitted a PR to deprecate these instead of removing them: https://github.com/saltstack/salt/pull/58741

Reply:
@major0 should we go ahead and close this PR then?